TwitchDownloaderCLI/Program.cs

```diff
@@ -24,13 +24,14 @@ private static void Main(string[] args)
                 config.HelpWriter = TextWriter.Null;
             });
 
-            var parserResult = parser.ParseArguments<VideoDownloadArgs, ClipDownloadArgs, ChatDownloadArgs, ChatUpdateArgs, ChatRenderArgs, FfmpegArgs, CacheArgs>(preParsedArgs);
+            var parserResult = parser.ParseArguments<TsMergeArgs, VideoDownloadArgs, ClipDownloadArgs, ChatDownloadArgs, ChatUpdateArgs, ChatRenderArgs, FfmpegArgs, CacheArgs>(preParsedArgs);
             parserResult.WithNotParsed(errors => WriteHelpText(errors, parserResult, parser.Settings));
 
             CoreLicensor.EnsureFilesExist(AppContext.BaseDirectory);
             WriteApplicationBanner((ITwitchDownloaderArgs)parserResult.Value, args);
 
             parserResult
+                .WithParsed<TsMergeArgs>(MergeTS.Merge)
```

Comment:
Why is it placed at the top? The previous order was more or less in order of importance.